packages/superset-ui-number-format/src/NumberFormatter.js

```diff
@@ -0,0 +1,35 @@
+import { ExtensibleFunction, isRequired } from '@superset-ui/core';
+
+export const PREVIEW_VALUE = 12345.432;
+
+export default class NumberFormatter extends ExtensibleFunction {
+  constructor({
+    formatName = isRequired('config.formatName'),
```

Comment:
this is used as a value in drop downs in the Superset App yeah? have mixed thoughts about calling it `formatName ` vs `formatValue` ... `formatValue` is a little obscure. maybe `formatId`? that would hint at uniqueness.

Reply:
Sounds good. I rename `formatName` to `id`. Thinking about doing the same for `ColorScheme`. Having both `name` and `label` is quite confusing.